docs/intro/quickstart/modeling.rst

```diff
@@ -0,0 +1,101 @@
+.. _ref_quickstart_modeling:
+
+=================
+Modeling the data
+=================
+
+.. edb:split-section::
+
+  The flashcards application has a simple data model, but it's interesting enough to get a taste of many of the features of the |Gel| schema language. You have a ``Card`` type that describes a single flashcard, which for now contains two required string properties: ``front`` and ``back``. Each ``Card`` belongs to a ``Deck``, and there is an explicit ordering to the cards in a given deck.
+
+  Looking at the mock data, you can see this structure in the JSON.
+
+  .. code-block:: typescript
+
+    interface Card {
+      front: string;
+      back: string;
+    }
+
+    interface Deck {
+      name: string;
+      description: string | null;
+      cards: Card[];
+    }
+
+.. edb:split-section::
+
+  Starting with this simple model, add these types to the ``default.gel`` schema file. As you can see, the types closely mirror the JSON mock data.
+
+  Also of note, the link between ``Card`` and ``Deck`` objects creates a "1-to-n" relationship, where each ``Deck`` object has a link to zero or more ``Card`` objects. When you query the ``Deck.cards`` link, the cards will be unordered, so the ``Card`` type needs an explicit ``order`` property to allow sorting them at query time.
+
+  .. code-block:: sdl-diff
+    :caption: dbschema/default.gel
+
+      module default {
+    +   type Card {
+    +     required order: int64;
+    +     required front: str;
+    +     required back: str;
+    +   };
+    +
+    +   type Deck {
+    +     required name: str;
+    +     description: str;
+    +     multi cards: Card {
+    +       constraint exclusive;
+    +     };
+    +   };
+      };
+
+.. edb:split-section::
+
+  Congratulations! This first version of the data model's schema is *stored in a file on disk*. Now you need to signal the database to actually create types for ``Deck`` and ``Card`` in the database.
+
+  To make |Gel| do that, you need to do two quick steps:
+
+  1. **Create a migration**: a file with a list of low-level instructions.
```

Comment:
I'd explain this a bit more clearly: a file containing a set of low level instructions that define how the database schema should change. It records any additions, modifications, or deletions to your schema in a way that the database can understand.

and something along the lines:
**Apply the migration**: This executes the migration file, instructing Gel to implement the recorded changes in the database. Essentially, this step updates the database structure to match your defined schema, ensuring that the Deck and Card types are created and ready for use.